linalgo/annotate/models.py

```diff
@@ -407,6 +409,23 @@ def __init__(
         self.annotator = Annotator(annotator)
         self.task = Task(task)
         self.reviewee = Annotator(reviewee)
-    
+
     def __repr__(self) -> str:
-        return f'Schedule::{self.type}::{self.status}'
\ No newline at end of file
+        return f'Schedule::{self.type}::{self.status}'
+
+
+@dataclass
+class Body:
+    text: str = "N/A"
```

Comment:
You can remove this class.

It is not helpful to constrain `Body` to have these attributes for all users.